buildSrc/build.gradle.kts

```diff
@@ -46,16 +46,29 @@ val mixinVersion: String = properties.getProperty("mixin.version")
 dependencies {
     implementation(gradleApi())
     implementation("gradle.plugin.org.cadixdev.gradle:licenser:0.6.1")
-    implementation("org.ajoberstar.grgit:grgit-gradle:5.2.1")
-    implementation("me.champeau.gradle:japicmp-gradle-plugin:0.4.0")
+    implementation("org.ajoberstar.grgit:grgit-gradle:5.2.2")
+    implementation("me.champeau.gradle:japicmp-gradle-plugin:0.4.2")
     implementation("com.github.johnrengelman:shadow:8.1.1")
-    implementation("org.jfrog.buildinfo:build-info-extractor-gradle:5.1.14")
+    implementation("org.jfrog.buildinfo:build-info-extractor-gradle:5.2.0")
     implementation("org.spongepowered:spongegradle-plugin-development:2.2.0")
     implementation("org.spongepowered:vanillagradle:0.2.1-20231105.223944-69")
-    implementation("net.minecraftforge.gradle:ForgeGradle:6.0.21")
+    val neoGradleVersion = "7.0.107"
+    implementation("net.neoforged.gradle:userdev:$neoGradleVersion")
+    implementation("net.neoforged.gradle:mixin:$neoGradleVersion")
     implementation("net.fabricmc:fabric-loom:$loomVersion")
     implementation("net.fabricmc:sponge-mixin:$mixinVersion")
     implementation("org.enginehub.gradle:gradle-codecov-plugin:0.2.0")
-    implementation("io.papermc.paperweight.userdev:io.papermc.paperweight.userdev.gradle.plugin:1.5.11")
-    implementation("org.spongepowered:mixingradle:0.7.38")
+    implementation("io.papermc.paperweight.userdev:io.papermc.paperweight.userdev.gradle.plugin:1.6.0")
+    constraints {
```

Comment:
This can be replaced by a maintained fork of shadowjar from here: https://github.com/Goooler/shadow
```kotlin
    implementation("io.github.goooler:shadow:8.1.7")
```

Reply:
I am hesitant to commit to any fork yet, especially after the xz fiasco. Gradle is going to have to pick some version of Shadow in the future and I'd rather reuse that decision when it's made. For now, this works without additional risk.